src/main/java/seedu/address/model/tag/UniqueTagList.java

```diff
@@ -0,0 +1,149 @@
+package seedu.address.model.tag;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.util.Iterator;
+import java.util.List;
+
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import seedu.address.model.tag.exceptions.DuplicateTagException;
+import seedu.address.model.tag.exceptions.TagNotFoundException;
+
+/**
+ * A list of tags that enforces uniqueness and does not allow nulls.
+ * A person is considered unique by comparing using {@code Tag#isSameTag(Tag)}. As such, adding and updating of
```

Comment:
I think can maybe change some of the comments here to say 'tag' instead of 'person'?

Reply:
Thanks for mentioning, i'll make the changes